docs/cluster_up_down.md

```diff
@@ -144,24 +148,28 @@ To stop your cluster, run:
 $ oc cluster down
 ```
 
-### Mac OS X with Docker Toolbox
+### Mac OS X with Docker Machine 
 
-1. Install [Docker Toolbox](https://www.docker.com/products/docker-toolbox) and ensure that it is functional.
-2. Install the oc binary using homebrew with: `brew install openshift-cli`
+1. Install [Docker for Mac](https://download.docker.com/mac/stable/21090/Docker.dmg) making sure you meet the [prerequisites](https://docs.docker.com/docker-for-mac/#/what-to-know-before-you-install).
+2. Install [VirtualBox for OSX Hosts](https://www.virtualbox.org/wiki/Downloads)
+3. Install the oc binary using homebrew with: `brew install openshift-cli`
 
    OR
 
    Download the OS X `oc` binary from [openshift-origin-client-tools-VERSION-mac.zip](https://github.com/openshift/origin/releases) and place it in your path.
 
    > Please be aware that the 'oc cluster' set of commands are only available in the 1.3+ or newer releases.
 
-3. Open Terminal and run
+4. Open Terminal and run
    ```
    $ oc cluster up --create-machine
    ```
 
-A Docker machine named `openshift` will be created using the VirtualBox driver and the OpenShift cluster
-will be started on it.
+   A Docker machine named `openshift` will be created using the VirtualBox driver and the OpenShift cluster
+   will be started on it.
+
+   > See [this gist](https://gist.github.com/adambkaplan/7e9b999cf25cfed4000b92b88a7d1941) to work around errors
+   downloading the boot2docker dependency from GitHub.
```

Comment:
you can include this content in a troubleshooting or appendix section of this doc rather than leaving it in a side-gist.